app/forms/work_form.rb

```diff
@@ -144,4 +146,10 @@ def mixed_materials_subtypes
     errors.add(:work_subtypes_mixed_materials,
                "#{WorkType::MINIMUM_REQUIRED_MIXED_MATERIAL_SUBTYPES} terms are the minimum allowed")
   end
+
+  def cited_contributor_presence
```

Comment:
This is the real change. Notice that it is a "standalone" error message in that it is not tied to a specific form field. This means that it will have to be displayed / cleared separately.